extension/popup.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Text Enhancer</title>
+  <link rel="stylesheet" href="styles.css">
```

Comment:
links to assets should be absolute links `<link rel="stylesheet" href="/styles.css">`